weft/assets.go

```diff
@@ -320,35 +326,40 @@ func loadAsset(file, prefix string) (*asset, error) {
 	return &a, nil
 }
 
-// initAssets loads all assets below dir into global maps.
-func initAssets(dir, prefix string) error {
+// InitAssets loads all assets below dir into global maps.
+func InitAssets(dir, prefix string) error {
 	var fileList []string
 
-	err := filepath.Walk(dir, func(path string, f os.FileInfo, err error) error {
-		fileList = append(fileList, path)
-		return nil
-	})
-	if err != nil {
-		return err
-	}
-
-	for _, v := range fileList {
-		fi, err := os.Stat(v)
+	assets = make(map[string]*asset)
+	assetHashes = make(map[string]string)
```

Comment:
I would move `L333-334` into `init` so that the maps are not recreated in subsequent calls of `InitAssets`